Spigot-Server-Patches/0552-Add-Option-to-disable-Witch-Transforms.patch

```diff
@@ -0,0 +1,34 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TheShermanTanker <tanksherman27@gmail.com>
+Date: Wed, 12 Aug 2020 02:06:35 +0800
+Subject: [PATCH] Add-Option-to-disable-Witch-Transforms
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+index e471e764935e2a89560de56959a782b02e5e8fe1..2a57a8a66a22a76853bb02e349a497c8765d77ec 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+@@ -72,6 +72,11 @@ public class PaperWorldConfig {
+         return config.getString("world-settings." + worldName + "." + path, config.getString("world-settings.default." + path));
+     }
+ 
++    public boolean witchConversion;
++    private void advancedSurvivalSettings() { //More may be added later
++    	witchConversion = getBoolean("advanced-survival-settings.witch-conversions", true);
```

Comment:
should be 4 spaces, not 1

Reply:
Wait, you meant that? I DID put 4 spaces though, why does it only show up as 1?